apiron/client.py

```diff
@@ -35,15 +35,38 @@ class ServiceCaller:
     A class for calling :mod:`services <apiron.service.base>`
     """
 
+    @staticmethod
+    def choose_response_format(endpoint, response, raw=False):
```

Comment:
This method name confuses me a bit. The method doesn't really choose a response format, but instead returns the response in a way determine by a choice made elsewhere. It may be more clear to get rid of this function and just the following instead of the one function call:

```python
response if return_raw_response else endpoint.format_response(response)
```

Reply:
I did exactly that originally; I split the method out for testability. If you take the current structure as given, is there a name that would make more sense?

Reply:
hmm I was thinking just `return response` but that might be confusing too given we could return a response object or text/json of the response. 

I can't think of a good name which might point to a code smell higher up. Tossing another idea out there, would it make sense to instead have a different method name like `call_raw()` or something like that to return the raw response and leave the current `call()` method as-is?

Reply:
That's an interesting idea; I'll mull over that a bit!

Reply:
I just tried creating a `call_raw()` function and it results in a lot of boilerplate given all of the parameters and some other strangeness around passing `*args` and `**kwargs` as an attempt to simplify the method signatures and invocation.

Reply:
I think duplication/boilerplate is ok if it serves the right purpose...if you want to screen share and talk about it we can!

Reply:
> I did exactly that originally; I split the method out for testability. If you take the current structure as given, is there a name that would make more sense?

How about `get_returnable_response`?

Reply:
Perhaps; we've opened a separate PR (#9) that solves the use case we wanted to cover for the moment. We should think deeper about this.